host/CMakeLists.txt

```diff
@@ -488,3 +488,5 @@ install(TARGETS oehostapp EXPORT openenclave-targets)
 install(TARGETS oehostverify EXPORT openenclave-targets
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/host
   COMPONENT OEHOSTVERIFY)
+
+include(OEHOSTMR/CMakeLists.txt)
```

Comment:
It would make sense to add a line `add_subdirectory(oehostmr)` here at the top-level CMakeLists.txt:
https://github.com/openenclave/openenclave/blob/99410b10b4782d33e76819bc681e651e96cd9739/CMakeLists.txt#L213

Reply:
@anakrish since oehostmr is part of host as oehostverify, is it better to stay under host directory? Actually the working directory for oehostmr is also host not host/oehostmr, because change working directory requires lots of change of cmake config and even head file reference. So right now include(OEHOSTMR/CMakeLists.txt) is a easy way to separate the config file. Let me know if I have some misunderstanding.